R/ccTable.R

```diff
@@ -73,155 +67,439 @@ create.cctable <- function(rec, freq, conf=NULL) {
     }
 
     cct <- ccTable(record=rec, conf=conf)
-    cct$create.table(freq)
+    cct$create_table(freq)
     return(cct)
 }
 
-#' get the dfilter
-#' @param dq can be either dqaulity table or torigin
-#' @param criterion should be a function to give T/F values of each entry.
-#' @export getfilter
-getfilter <- function(dq, criterion) {
-    if (ncol(dq) > 2) {
-        keys <- dq[, c("site", "episode_id"), with=FALSE]
-        dq[, c("site", "episode_id"):=NULL]
-        # updating range entry with true/false values
-        dq <- dq[, Map(criterion, .SD, names(.SD))]
-        # adding site and episode_id columns.
-        entry <- data.table(keys, dq)
-        episode <- entry[, 
-                         all(unlist(.SD), na.rm=TRUE), 
-                         by=c("site", "episode_id")]
-        setnames(episode, c("site", "episode_id", "select_index"))
-        return(list(entry=entry, episode=episode))
-    }
-    else return(NULL)
-}
+#' Create a ccTable object
+#'
+#' This is a member function of ccTable-class. Using create_cctable is a safer and 
+#' easier way to create the ccTable. See create_cctable. 
+#' @name ccTable_create_cctable
+NULL
```

Comment:
what does this `NULL` does?

Reply:
Because refclass doesn't have a proper way to document. NULL means it is an orphan document which doesn't link to anywhere.